src/Components/dashboard/operatorsanddriver/DriverAndOperator.jsx

```diff
@@ -11,7 +11,7 @@ import Box from "@mui/material/Box";
 import Tab from "@mui/material/Tab";
 import TabContext from "@mui/lab/TabContext";
 import TabList from "@mui/lab/TabList";
-import Skeleton from "react-loading-skeleton";
+
```

Comment:
Don't we need a loading skeleton here? I can see you have removed it.

Reply:
We don't need loading skeleton here as we decided to use component, so we need this import in the component that will be created.